docs/tutorials/typography.rst

```diff
@@ -388,25 +388,32 @@ Drawing letters to the screen becomes more engaging when used in combination wit
 
    letters = ""
    back = 102
+   fil = 0
+
 
    def setup():
       size(100, 100)
       text_align("CENTER")
-      text_size(16)
+      # text_size(16)
```

Comment:
I'd suggest either use it or don't, don't comment it out. If you don't like the size but want to use it, make it 10 or 12.

Reply:
This is because I saw another example with "text_size()" comment it out, the example is in "Typography" tutorial, the last example have that function comment it out and I think it is related to the fact that the current default font is bitmap, so the text_size function is not yet implemented for that format.

Reply:
I'm going to delete the "text_size" since is not implemented yet for bitmap fonts and because using it require make the font a vector and the example became a little to complex